src/vaadin-dialog-resizable-mixin.html

```diff
@@ -193,6 +193,40 @@
         window.removeEventListener('touchmove', this._resizeListeners.resize[direction]);
         window.removeEventListener('mouseup', this._resizeListeners.stop[direction]);
         window.removeEventListener('touchend', this._resizeListeners.stop[direction]);
+        const dim = this._getResizeDimensions();
+        this.dispatchEvent(
+          new CustomEvent('resize', {
+            detail: {
+              width: dim.overlay.width,
+              height: dim.overlay.height,
+              contentWidth: dim.content.width,
+              contentHeight: dim.content.height
+            }
+          })
+        );
+      }
+
+      _getResizeDimensions() {
+        const overlay = this.$.overlay;
+        const overlayStyles = getComputedStyle(overlay.$.overlay);
+        const contentStyles = getComputedStyle(overlay.$.content);
+        const verticalPadding =
```

Comment:
Now the width and height are obtained with different means. Would the following work? It's a bit verbose though; if you know a way to shorten it, by all means.
```
content.setAttribute('style', 'position: absolute; top: 0; right: 0; bottom: 0; left: 0; box-sizing: content-box; height: auto;');
const {width, height} = getComputedStyle(content);
content.removeAttribute('style');
```

Reply:
Oh nice, using absolute position is definitely quicker than having to calculate the padding. I also keep forgetting about destructuring assignment; should be using these new features more now with chromium edge.

Reply:
I refactored the code now